mod/message.php

```diff
@@ -262,8 +262,9 @@ function message_content(App $a) {
 				//	);
 				//}
 
-				if($r)
+				if ($r) {
```

Comment:
Similar with here, DELETE is the statement (and a typical candiate is). I'm not sure if `dbm::is_result()` can be really used here.